src/Sarif/Schemata/sarif-schema.json

```diff
@@ -20,7 +20,8 @@
     "runs": {
       "description": "The set of runs contained in this log file.",
       "type": "array",
-      "minItems": 1,
+      "minItems": 0,
+      "uniqueItems": false,
```

Comment:
>uniqueItems [](start = 7, length = 11)

basically now, any array with minItems == 0 which are not 'anyOf + null' as this one is, will have a default value of "[]". Yes?

Reply:
Yes, I can't think of any exceptions to that.

---
In reply to: [245121606](https://github.com/Microsoft/sarif-sdk/pull/1201#discussion_r245121606) [](ancestors = 245121606)